Assets/MixedRealityToolkit.Providers/UnityAR/Editor/UnityARCameraSettingsProfileInspector.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using Microsoft.MixedReality.Toolkit.Editor;
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
```

Comment:
Do you have an issue for AR Foundation documentation?

Reply:
should be #6346